pymtl3/passes/sverilog/translation/test/SVTranslator_L2_cases_test.py

```diff
@@ -1148,6 +1148,37 @@ def upblk():
 """
   do_test( a )
 
+def test_lambda_connect( do_test ):
+  class A( Component ):
+    def construct( s ):
+      s.in_ = InPort( Bits32 )
+      s.out = OutPort( Bits32 )
+      s.out //= lambda: s.in_ + Bits32(42)
+  a = A()
+  a._ref_src = \
+"""
+module A
+(
+  input logic [0:0] clk,
+  input logic [31:0] in_,
+  output logic [31:0] out,
+  input logic [0:0] reset
+);
+
+  // PYMTL SOURCE:
+  //
+  // def _lambda__s_out():
```

Comment:
Can you copy paste the actual output here?

Reply:
done